lifecycle-operator/apis/lifecycle/v1alpha3/keptnappversion_types.go

```diff
@@ -111,7 +111,6 @@ type WorkloadStatus struct {
 }
 
 // +kubebuilder:object:root=true
-// +kubebuilder:storageversion
```

Comment:
as long as v1beta1 is not used in the controller logic, the storage version should not be changed. So let's keep the storageversions unchanged for now

Reply:
done